content/docs/contributor/anteaterapi/setup.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Setting up
+description: Setting up to contribute to Anteater API
```

Comment:
I feel this description is a bit redundant. Perhaps something along the lines of "Setting up (Anteater API locally / an Anteater API environment locally) for contributing."

Honestly fine as is though.